.eslintrc

```diff
@@ -13,6 +13,7 @@
 		"jQuery": true,
 		"wp": false
 	},
+	"extends": 'wordpress',
```

Comment:
Since we are using the WordPress config for styles, let's also use the config for JavaScripts.